src/wp-includes/media.php

```diff
@@ -1958,6 +1990,10 @@ function wp_image_use_alternate_mime_types( $image, $context, $attachment_id ) {
 
 			$image = str_replace( $src_filename, $metadata['sources'][ $target_mime ]['file'], $image );
 
+			if ( true !== $_wp_image_mime_fallback_should_load ) {
```

Comment:
Using the global looks good, but there are still conditions missing here:
* We should only do this if the `$context` is "the_content".
* We should only do this if the `$target_mime` is "image/webp".
* We don't need to check however whether the value is already set to `true`. It's not bad to do this, but it's an unnecessary condition.

Reply:
Code updated with the conditions.